### ISSUE TITLE
Pre-trained model size, *.params file is 435.4M

### ISSUE THREAD

**Issue by clhne** [deepinsight/insightface#545]
Hi @nttstar 
The pre-trained model of .params (model-r100-ii) is 249M, but my .params file size is 435.4M, why it's so large, and which may cause this?
config:
batch_size =32
run script: 
python -u train_softmax.py --network r100 --loss-type 4 --margin-m 0.5 --data-dir /workspace/dataset/ms1m/third_party/faces_ms1m_112x112 --prefix ../model-r100

Thanks.

**Comment by nttstar** [comment]
Try using recognition/train.py which will remove last fc7 automatically(with an option).

**Comment by clhne** [comment]
Yeah, after i use recognition/train.py, the *.params file size is 249M, cool!
But, I find your pre-trained model is 3.9M：
![image](https://user-images.githubusercontent.com/11884005/53321690-21296180-3914-11e9-983a-21d24b56a220.png)

so, how to fix this? and I want know the details you do this.
Thanks.

**Comment by Isaver23** [comment]
@nttstar can we still get the original models whose 'fc7' layer aren't deleted yet? many thanks!

**Comment by Isaver23** [comment]
Hi @clhne, I wonder can you give a copy of the 435.4M r100 model? Many thanks!

**Comment by clhne** [comment]
The file is too large, in which way I can give you?
I'm wonder why you need the pretrained model?

**Comment by Isaver23** [comment]
Hi @clhne , I'm trying to finetune my r100 model with emore datasets, so I need the pretrained fc7 layers as my baseline. Any way is ok for me, maybe BaiduYun, Dropbox, email (My email address is in my github homepage~), or even QQ is ok for me! Thanks a lot for your help!

**Comment by SueeH** [comment]
> Hi @clhne , I'm trying to finetune my r100 model with emore datasets, so I need the pretrained fc7 layers as my baseline. Any way is ok for me, maybe BaiduYun, Dropbox, email (My email address is in my github homepage~), or even QQ is ok for me! Thanks a lot for your help!

for finetuning, fc7 can initialize randomly, it’s not necessary to copy author‘s fc7 params.
